build.sbt

```diff
@@ -0,0 +1,39 @@
+val sbtPluginSettings = ScriptedPlugin.scriptedSettings ++ Seq(
+    organization := "org.scala-native",
+    version := "0.1-SNAPSHOT",
+    sbtPlugin := true,
+    scalaVersion := "2.10.6",
+    scriptedLaunchOpts += "-Dplugin.version=" + version.value
+  )
+
+lazy val `sbt-cross-project` =
+  project
+    .in(file("."))
+    .aggregate(sbtScalaJsCross, sbtCross)
+    .dependsOn(sbtScalaJsCross, sbtCross)
+
+lazy val sbtScalaJsCross =
+  project
+    .in(file("sbt-scalajs-cross"))
+    .settings(sbtPluginSettings)
+    .settings(
+      moduleName := "sbt-scalajs-cross",
+      addSbtPlugin("org.scala-js" % "sbt-scalajs" % "0.6.13" % "provided")
```

Comment:
> It's a way to enfore not removing sbt-scalajs in your plugins.sbt ;-).

It's mostly a way to throw `LinkageError`s at people's face rather than compile errors. IMO this does not improve the situation.

People will not remove sbt-scalajs from their builds because they want to upgrade Scala.js anyway.